cns/service/main.go

```diff
@@ -1059,17 +1068,15 @@ func reconcileInitialCNSState(ctx context.Context, cli nodeNetworkConfigGetter,
 			return errors.Wrapf(err, "failed to convert NNC status to network container request, "+
 				"assignmentMode: %s", nnc.Status.NetworkContainers[i].AssignmentMode)
 		}
-		// Get previous PodInfo state from podInfoByIPProvider
-		podInfoByIP, err := podInfoByIPProvider.PodInfoByIP()
-		if err != nil {
-			return errors.Wrap(err, "provider failed to provide PodInfoByIP")
-		}
 
-		// Call cnsclient init cns passing those two things.
-		if err := restserver.ResponseCodeToError(ncReconciler.ReconcileNCState(ncRequest, podInfoByIP, nnc)); err != nil {
-			return errors.Wrap(err, "failed to reconcile NC state")
-		}
+		ncReqs = append(ncReqs, ncRequest)
 	}
+
+	// Call cnsclient init cns passing those two things.
+	if err := restserver.ResponseCodeToError(ncReconciler.ReconcileNCState(ncReqs, podInfoByIP, nnc)); err != nil {
+		return errors.Wrap(err, "failed to reconcile NC state")
```

Comment:
aligning with my other comments, this should either say NNC state or CNS ipam state